storybook/storybook-css/src/components/UnorderedList/UnorderedList.docs.mdx

```diff
@@ -9,3 +9,10 @@ import README from "../../../../../packages/css/src/unordered-list/README.md?raw
 <Primary />
 
 <Controls />
+
+### No Marker
+
+Voor een lijst zonder marker gebruik je de `no-marker` modifier.
+Deze wordt gebruikt voor onder andere de navigatie in de header of footer maar ook voor zoekresultaten.
```

Comment:
How do you like this?

```suggestion
### Zonder opsommingsteken

Gebruik dit component niet alleen voor tekstuele opsommingen, maar ook voor lijsten van grotere secties zoals een overzicht van artikelen of een verzameling zoekresultaten. Semantisch zijn dit immers ook lijsten.

Gebruikers van schermlezers krijgen op deze manier ook informatie over de lengte van de lijst en het volgnummer van het item waarop ze focus zetten.

Je maakt zo’n ‘onzichtbare’ lijst via de `no-marker` modifier.
```

Reply:
The last piece can become ‘de `noMarker` prop’ in the docs for the React component.